src/irqhandler.cpp

```diff
@@ -52,6 +52,14 @@ void irqHandler(void *pvParameters) {
     }
 #endif
 
+// E-Ink display needs refresh?
+#ifdef HAS_E_PAPER_DISPLAY
+if (InterruptStatus & E_PAPER_DISPLAY_IRQ) {
```

Comment:
Where is E_PAPER_DISPLAY_IRQ set? Can't find a trigger. It looks like display is only refreshed when the button is pressed, but not by a cyclic IRQ. Thus, we don't need this?

Reply:
Yes you are right. I did not completely understand what this was for and thought it might be important for other things and also thought you might want it at a later time for automatic screen updates (which we dont need in our project)